src/components/App/Dashboard/Balances/Balances.js

```diff
@@ -1,37 +1,63 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { capitalize } from 'lodash';
-import { ScrollView } from 'react-native';
-import AmountItem from 'components/shared/AmountItem';
-import { toFixed } from 'utils';
 
-import { Container } from './styles';
+import { FlatList } from 'react-native';
+import { BalanceItem } from 'components/shared/Financialtems';
+import { getCurrencyIconUri } from 'utils/currency-icon-uri';
 
 class Balances extends Component {
   static propTypes = {
-    balances: PropTypes.array,
-    change: PropTypes.object,
+    balances: PropTypes.array.isRequired,
+    removeUserCurrency: PropTypes.func.isRequired,
   };
 
   render() {
-    const { balances, change } = this.props;
-
+    const { balances } = this.props;
+    const keyedBalances = balances
+      .map(balance => ({
```

Comment:
Same as comments as in `CurrenciesList.js:`[23](https://github.com/mobius-network/wallet/pull/68/files#diff-c6a5e2113764e4c4d5aa6374850f1efdR23) and `CurrenciesList.js:`[29](https://github.com/mobius-network/wallet/pull/68/files#diff-c6a5e2113764e4c4d5aa6374850f1efdR29)